### PR TITLE
open source modules added, with dns and acm modules

### PR DESCRIPTION
**Pull request by paulroesch**
tested successfully access to mydb from asg instances using mysql cmd

### CODE REVIEW

File: level1/main.tf [hunk 1]
```diff
@@ -1,10 +1,26 @@
+# module "vpc" {
```
Comment (geoffrey1330) [inline level1/main.tf]: could you please remove commented code from file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level1/main.tf (hunk 1 above)
Comment (paulroesch) [reply level1/main.tf]: done

File: level2/rds.tf [hunk 2]
```diff
@@ -10,14 +10,64 @@ locals {
...
+
+  engine               = "mysql"
+  engine_version       = "5.7"
+  family               = "mysql5.7"
+  major_engine_version = "5.7"
+  instance_class       = "db.t3.micro"
+  allocated_storage    = 5
+  identifier           = var.env_code
+
+  db_name                             = "mydb"
```
Comment (geoffrey1330) [inline level2/rds.tf]: could you please format your code.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level2/rds.tf (hunk 2 above)
Comment (paulroesch) [reply level2/rds.tf]: ok done

File: level2/load-balancer.tf [hunk 3]
```diff
@@ -1,7 +1,73 @@
...
+  version = "5.1.0"
 
-  vpc_id           = data.terraform_remote_state.level1.outputs.vpc_id
-  env_code         = var.env_code
-  public_subnet_id = data.terraform_remote_state.level1.outputs.public_subnet_id
+  name        = "${var.env_code}-external"
+  description = "Security group for allow http traffic to ELB and egress traffic to targets"
+  vpc_id      = data.terraform_remote_state.level1.outputs.vpc_id
+
+  ingress_with_cidr_blocks = [
```
Comment (geoffrey1330) [inline level2/load-balancer.tf:52]: you didn't consider use of https and certificate manager to handle https request.
could you please add this suggestion to your PR.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level2/load-balancer.tf (hunk 3 above)
Comment (paulroesch) [reply level2/load-balancer.tf:52]: added now, can you please review. thank you

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**